### ISSUE TITLE
Move all language definition into the core VS Code product

### ISSUE THREAD

**Issue by davidanthoff** [julia-vscode/julia-vscode#1997]
In particular, I think we can move [this](https://github.com/julia-vscode/julia-vscode/blob/3337b8fec2d171e05f435a3a4a10c619e2b534ed/src/extension.ts#L52) over into https://github.com/microsoft/vscode/blob/main/extensions/julia/language-configuration.json, right?

I actually don't understand right now why that is in the `activate` function rather than in the declarative file for setting this stuff...

**Comment by pfitzseb** [comment]
Yeah, I remember looking at that and wondering why it wasn't in language-configuration.json. I feel like I'd moved it there at some point, but apparently never pushed that anywhere...
We could add more elaborate (function based) indentation rules in `activate`, but that shouldn't stop us from moving what we have out.
